### ISSUE TITLE
[Feature] PDF Support

### ISSUE THREAD

**Issue by hassaffe** [stashapp/stash#1006]
It would be good if you could manage PDF files under a menu item Ebooks. One could select an image as a preview. And then open the pdf as normal in the browser. I don't mean text files like in the other post.

**Comment by WithoutPants** [comment]
Similar to #14

**Comment by Arcturuss** [comment]
> Similar to [#14](https://github.com/stashapp/stash/issues/14)

not really. PDFs are pre-formatted and could be treated more like images.
